### DISCUSSION TITLE
UI-UX is Terrific - Here's what I like and don't like

### DISCUSSION THREAD

**Discussion by drdancm** [anapnoe/stable-diffusion-webui-ux#202] [General]
Overall I really like UI-UX !

PROS: 
1 It seems to process things faster than A1111, and I have less or no problems running similar extensions without sabotage. For example FSwap programs Reactor and Face Swap Lab. FSLab does a great blending job with multiple face files but does a poorer job with the actual swap than Reactor. Reactor produces clearer and sharper faces. so it is wonderful that the two programs get along !

2 Installs more of the models and requisites etc. than A1111 automatically which makes it work more efficiently and much more easily for people who are not experts. This was especially true with Reactor. Tons of issues with getting  this extension to run properly have to do with not installing the requisites.

CONS:

1 If you are familiar with the A1111 layout, you will have a bit of a learning curve to find where things are. Once you find them and remember them you will be OK. Some of the placement of Elements are, once you are familiar, better than A1111.

2 The Icons are impossible to figure out on their own without the text. It's not clear whether any icons can ever work on their own, but somehow, this slows down the time it takes to become more proficient in using the package.
Perhaps it would be useful to have the Names or Maybe First two/3 letters of the names of the feature inside the icons.

For example, along top left icon for Extensions is a US AC electric plug. If it had Ext inside the Icon it would be far better. Maybe twice the width with the letters? A plug is not an extension, although I can imagine an extension plug, but that does not look like an simple USA AC electric plug. Yes, I understand Icons are helpful because they are supposedly universal (language differences), but let's face it English names are pretty much standard for all of the things we are talking about.  Abstract things are pretty much impossible to convey with Icons.

**Comment by anapnoe** [comment]
Hi you are welcome 
you can enable labels for all icons 
options are located in settings Anapnoe UI UX enable labels for tabs, pages, and aside view respectfully
